examples/minnietwitter/src/main/java/amino/run/appexamples/minnietwitter/MinnieTwitterMain.java

```diff
@@ -142,46 +142,159 @@ private static void ExecuteSingleUserDemo(InetSocketAddress hostAddr, InetSocket
     }
   }
 
-  private static void ExecuteFullDemo(InetSocketAddress hostAddr, InetSocketAddress omsAddr) {
+  /**
+   * Create an instance of the TwitterManager Microservice
+   * @param oms
+   * @return
+   * @throws RemoteException
+   * @throws MicroServiceCreationException
+   * @throws MicroServiceNotFoundException
+   */
+  private static TwitterManager createTwitterManager(Registry oms, String microServiceName) throws RemoteException, MicroServiceCreationException, MicroServiceNotFoundException, MicroServiceNameModificationException {
+    /* Get Twitter and User Manager */
+    MicroServiceSpec spec =
+            MicroServiceSpec.newBuilder()
+                    .setLang(Language.java)
+                    .setJavaClassName("amino.run.appexamples.minnietwitter.TwitterManager")
+                    .create();
```

Comment:
It seems there was no DMs used. Doesn't need to be done in this PR.

Reply:
Yes, 100% agreed.  I think I'll add them in this PR.